gdist.py

```diff
@@ -0,0 +1,76 @@
+from ctypes import POINTER, CDLL, c_int, c_uint, c_double
+import glob
+
+import numpy as np
+import scipy
+
+libfile = glob.glob('build/*/gdist*.so')[0]
```

Comment:
I suspect this needs to be a little more robust, but you could come back to it once it's compiling across platforms.  It should be `.dylib` on macOS, `.dll` on Windows, for example.

Reply:
Yup.
Note to self: Add a macOS job in Travis.

Reply:
@maedoc Can you please point me to some resources/examples for using ctypes with windows?